Make initialize_docker fails

Setting up addons-server via docker on a new machine, the `make initialize_docker` command fails with:

```
python manage.py import_prod_versions
13:20:35 z.startup:INFO Set RECURSION_LIMIT to 10000 :/code/src/olympia/core/apps.py:82
13:20:36 raven.base.Client:DEBUG Configuring Raven for host: <raven.conf.remote.RemoteConfig object at 0x3d96390> :/usr/lib/python2.7/site-packages/raven/base.py:255
13:20:36 raven.base.Client:INFO Raven is not configured (logging is disabled). Please see the documentation for more information. :/usr/lib/python2.7/site-packages/raven/base.py:202
Import versions for firefox
Import versions for thunderbird
Import versions for android
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/usr/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/code/src/olympia/applications/management/commands/import_prod_versions.py", line 18, in handle
    app = APP_GUIDS[codes[i].text]
KeyError: '{a23983c0-fd0e-11dc-95ff-0800200c9a66}'
make[1]: *** [initialize_db] Error 1
make[1]: Leaving directory `/code'
make: *** [initialize_docker_inner] Error 2
make: *** [initialize_docker] Error 2
```

Possibly caused by: https://github.com/mozilla/addons-server/commit/2c487211d24c5a61a325de3ec6dbca676d255e6e

--- a/src/olympia/applications/management/commands/import_prod_versions.py
+++ b/src/olympia/applications/management/commands/import_prod_versions.py
@@ -14,8 +14,12 @@ class Command(BaseCommand):
         doc = PyQuery(
             url='https://addons.mozilla.org/en-US/firefox/pages/appversions/')
         codes = doc('.prose ul li code')
-        for i in range(0, codes.length, 2):
-            app = APP_GUIDS[codes[i].text]
+        for i in range(0, len(codes), 2):
+            try:
+                app = APP_GUIDS[codes[i].text]
+            except KeyError:
+                # Unknown app, ignore.
+                continue
             log('Import versions for {0}'.format(app.short))
             versions = codes[i + 1].text.split(', ')
             for version in versions:

--- a/src/olympia/applications/tests.py
+++ b/src/olympia/applications/tests.py
@@ -85,10 +85,13 @@ class TestCommands(TestCase):
         assert len(AppVersion.objects.filter(
             application=amo.FIREFOX.id, version=new_version)) == 1
 
-    @mock.patch('olympia.applications.management.commands.import_prod_versions.PyQuery', spec=True)
+    @mock.patch('olympia.applications.management.commands.import_prod_versions'
+                '.PyQuery', spec=True)
     def test_import_prod_versions(self, pyquery_mock):
-        assert not AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.0').no_cache().exists()
-        assert not AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.*').no_cache().exists()
+        assert not AppVersion.objects.filter(
+            application=amo.FIREFOX.id, version='53.0').no_cache().exists()
+        assert not AppVersion.objects.filter(
+            application=amo.FIREFOX.id, version='53.*').no_cache().exists()
 
         # Result of PyQuery()
         MockedDoc = mock.Mock()
@@ -106,5 +109,7 @@ class TestCommands(TestCase):
 
         call_command('import_prod_versions')
 
-        assert AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.0').no_cache().exists()
-        assert AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.*').no_cache().exists()
+        assert AppVersion.objects.filter(
+            application=amo.FIREFOX.id, version='53.0').no_cache().exists()
+        assert AppVersion.objects.filter(
+            application=amo.FIREFOX.id, version='53.*').no_cache().exists()

--- a/src/olympia/applications/tests.py
+++ b/src/olympia/applications/tests.py
@@ -1,4 +1,5 @@
 import json
+import mock
 import tempfile
 
 from django.core.management import call_command
@@ -83,3 +84,27 @@ class TestCommands(TestCase):
 
         assert len(AppVersion.objects.filter(
             application=amo.FIREFOX.id, version=new_version)) == 1
+
+    @mock.patch('olympia.applications.management.commands.import_prod_versions.PyQuery', spec=True)
+    def test_import_prod_versions(self, pyquery_mock):
+        assert not AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.0').no_cache().exists()
+        assert not AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.*').no_cache().exists()
+
+        # Result of PyQuery()
+        MockedDoc = mock.Mock()
+        pyquery_mock.return_value = MockedDoc
+
+        # Result of PyQuery()('selector'). Return 2 applications, one with a
+        # valid guid and one that is garbage and should be ignored.
+        MockedDocResult = [
+            mock.Mock(spec=[], text='lol'),
+            mock.Mock(spec=[], text='some versions...'),
+            mock.Mock(spec=[], text='{ec8030f7-c20a-464f-9b0e-13a3a9e97384}'),
+            mock.Mock(spec=[], text='53.0, 53.*'),
+        ]
+        MockedDoc.return_value = MockedDocResult
+
+        call_command('import_prod_versions')
+
+        assert AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.0').no_cache().exists()
+        assert AppVersion.objects.filter(application=amo.FIREFOX.id, version='53.*').no_cache().exists()

